### PR TITLE
Settings component + units for input field + powergate 0.2.0 syntax updates

### PR DESCRIPTION
**Pull request by martinalong**
- Settings component mostly completed 
    - Still waiting on small detail: what unit of measurement "threshold" is in
    - Remove unused tooltips if nothing to add, and polish the copywriting
    - Trusted/excluded miners list + country codes list to come
- Standardized "full" prop for inputs / description group / dropdowns

image

### CODE REVIEW

**Comment by martinalong** [comment]
- Added support for units in input fields 
- Minor bug fix in settings component. Previously didn't allow for "regular" (non async) passing of props. Now it does

image

**Comment by martinalong** [comment]
- Syntax updates in make-storage-deal.js: `pushConfig()` -> `pushStorageConfig()` and `addToHot()` -> `stage()` (I tested it and it works)
- All syntax should be compatible with powergage 0.2.0 now

**Comment by jimmylee** [comment]
LGTM, This is GREAT work. NICE one Martina :)

File: pages/experiences/filecoin-settings.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  }
+
+  render() {
+    return (
+      <FilecoinSettings 
+        autoApprove={this.state.autoApprove}
+        defaultStorageConfig={this.state.defaultStorageConfig} 
+        addrsList={this.state.addrsList} 
+        onSave={this._handleSave} 
+        />
```
Comment (jimmylee) [inline pages/experiences/filecoin-settings.js:83]: Nit: Not going to make you fix this but formatting is slightly off here.

**Pull request merged**